indico/modules/events/persons/controllers.py

```diff
@@ -151,17 +148,38 @@ def _checkParams(self, params):
 
     def _process(self):
         person_ids = request.form.getlist('person_id')
-        recipients = {p.email for p in self._find_event_persons(person_ids) if p.email}
-        form = EmailEventPersonsForm(person_id=person_ids, recipients=', '.join(recipients))
+        recipients = self._find_event_persons(person_ids)
+        if request.args.get('not_invited_only') == '1':
+            recipients = {p for p in recipients if p.invited_dt is None}
+        default_body = ''
+        default_subject = ''
+        disable_until_change = True
+        if request.args.get('no_account') == '1':
```

Comment:
`no_account = request.args.get('no_account') == '1'` and use it here and also below. Right now you use two different checks for the same argument which aren't even equal in behavior